circle.yml

```diff
@@ -4,6 +4,7 @@ machine:
 
 dependencies:
   cache_directories:
+    - "content"
```

Comment:
Why do we need to cache the content directory? Is this where the sqlite files are written?